Block/Tracking.php

```diff
@@ -22,24 +24,43 @@ class Tracking extends Template
 
     protected $_storeManager;
 
+    protected $_customerSession;
+
     public function __construct(
         Context               $context,
         FormKey               $formKey,
         Currency              $_currency,
-        StoreManagerInterface $_storeManager
+        StoreManagerInterface $_storeManager,
+        Session               $_customerSession
     )
     {
         parent::__construct($context);
         $this->formKey = $formKey;
         $this->_currency = $_currency;
         $this->_storeManager = $_storeManager;
+        $this->_customerSession = $_customerSession;
+    }
+
 
+    /**
+     * @return string
+     */
+    public function getCustomerEmail()
+    {
+        $email = "";
+        try {
+            $customerData = $this->_customerSession->getCustomer();
+            $email = $customerData->getEmail();
+        } catch (Exception) {
```

Comment:
Hey @stubbedev: you made a similar mistake again like I fixed earlier in https://github.com/clerkio/clerk-magento2/pull/112

This leads to errors like: `class Clerk\Clerk\Block\Exception not found`

Could you fix it again, by using `\Exception` instead of `Exception`? And pay a bit more attention to this in the future, thanks! 🙂